src/test/java/org/cactoos/io/InputOfTest.java

```diff
@@ -323,44 +322,15 @@ void makesDataAvailable() throws Exception {
     }
 
     @Test
-    // @checkstyle MethodBodyCommentsCheck (50 lines)
-    void readsSecureUrlContent() throws Exception {
-        final TrustManager[] managers = {
-            new X509TrustManager() {
-                @Override
-                public X509Certificate[] getAcceptedIssuers() {
-                    return new X509Certificate[0];
-                }
-
-                @Override
-                public void checkClientTrusted(
-                    final X509Certificate[] cert, final String arg) {
-                    // nothing to do
-                }
-
-                @Override
-                public void checkServerTrusted(
-                    final X509Certificate[] cert, final String arg) {
-                    // nothing to do
-                }
-            },
-        };
-        final SSLContext context = SSLContext.getInstance("TLS");
-        context.init(null, managers, new SecureRandom());
-        HttpsURLConnection.setDefaultSSLSocketFactory(
-            context.getSocketFactory()
+    void readsUrlContent() throws Exception {
+        final Text data = new TextOf("hello");
+        new FtRemote(new TkFork(new FkRegex("/", new TkText(data.asString())))).exec(
+            uri -> new Assertion<>(
+                "Must read bytes from HTTPS URL",
+                new TextOf(new InputOf(uri)),
```

Comment:
@andreoss this test seems to exist to test that it works with https, but I don't think that `FtRemote` is serving https at all…

Reply:
@victornoel Than I suggest removing this test completely than. The code under test doesn't depend on protocol at all

Reply:
@andreoss ok, that makes sense, and since the code is relying on Java's std lib for http requests, there is no big reason for testing it. Thanks for looking into it